libtransmission/utils.h

```diff
@@ -7,15 +7,19 @@
 
 #include <algorithm>
 #include <cctype>
+#include <chrono>
 #include <cstdint> // uint8_t, uint32_t, uint64_t
 #include <cstddef> // size_t
 #include <ctime> // time_t
 #include <optional>
 #include <string>
 #include <string_view>
+#include <thread>
 #include <type_traits>
 #include <vector>
 
+using namespace std::chrono_literals;
```

Comment:
Is it good practice?
Feels like `using namespace ...` in header file pollutes consumer namespace.

Reply:
Well,  consistent usage of  `std::chrono::duration` literals through the whole code base might be a notable exception.

Reply:
@nevack is write; this goes against [this cpp core guideline](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#sf7-dont-write-using-namespace-at-global-scope-in-a-header-file) and will generate a new sonarcloud warning when merged.  Let's try removing this from the header and see how cumbersome it gets; we can always add it back in later if the cc-level cleanup is a burden.

Reply:
OK